public/static/docs/command-reference/get.md

```diff
@@ -12,17 +12,18 @@ Download a file or directory from any <abbr>DVC project</abbr> or Git repository
 usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to a file or directory within a DVC repository.
+  url                   Location of DVC project or Git repository to download
+                        from
+  path                  Path to a file or directory within the project or
+                        repository
 ```
 
 ## Description
 
-Provides an easy way to download files or directories tracked in any <abbr>DVC
-repository</abbr>, both by Git (e.g. source code) or DVC (e.g. datasets, ML
-models). The file or directory in path is copied to the current working
-directory. (For remote URLs, it works like downloading with wget, but supporting
-DVC <abbr>data artifacts</abbr>.)
+Provides an easy way to download files or directories tracked in a, both by Git
```

Comment:
`tracked in a,` ? something is missing here?

Reply:
Maybe use "...tracked in any <abbr>DVC project</abbr> (e.g. datasets, ML models) or Git repository (e.g. source code). The file or directory..."